plash_cli/core.py

```diff
@@ -106,23 +97,32 @@ def _deps(script: bytes | str) -> dict | None:
         return '\n'.join(tomllib.loads(content)['dependencies'])
     else: return None
 
-# %% ../nbs/00_core.ipynb 15
```

Comment:
This name is expected in plash as it imports plash_cli so we can't change it here unless we also want to change it in the plash server. For this change, I think it is fine to keep the same name to be consistent